doc/api/url.md

```diff
@@ -115,8 +115,10 @@ No decoding of the `path` is performed.
 ### urlObject.query
 
 The `query` property is either the "params" portion of the query string (
-everything *except* the leading ASCII question mark (`?`), or an object
-returned by the [`querystring`][] module's `parse()` method:
+everything *except* the leading ASCII question mark (`?`) ), or an object
```

Comment:
How about we rephrase that first sentence to make it more concise and eliminate the nested parentheses:

```
The `query` property is either the query string without the leading question
mark (`?`) or an object returned by the [`querystring`][] module's `parse`
method.
```